main/webapp/modules/core/styles/project/sidebar.css

```diff
@@ -254,4 +254,5 @@ textarea.history-operation-json {
 
 .history-operation-json-error {
   color: var(--error-red);
+  max-width: 750px;
```

Comment:
`max-width: calc(800px - var(--padding-looser));` in case you want to make it pixel perfect.